assets/base/i18n/en.json

```diff
@@ -3304,6 +3304,7 @@
   "user.settings.notifications.email.disabled_long": "Email notifications have not been enabled by your System Administrator.",
   "user.settings.notifications.email.everyHour": "Every hour",
   "user.settings.notifications.email.everyXMinutes": "Every {count, plural, one {minute} other {{count, number} minutes}}",
+  "user.settings.notifications.email.fifteenMinutes": "Every 15 minutes",
```

Comment:
I think this needs a `mobile.` prefix to be able to be changed without making a PR to the web app repo

Reply:
It's now updated. Thanks!